spec/requests/api_v1_collection_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe Hyku::API::V1::CollectionController, type: :request, clean: true, multitenant: true do
+  let(:account) { create(:account) }
+  let(:collection_type) { create(:collection_type, brandable: true) }
+  let(:collection_type_gid) { collection_type.to_global_id.to_s }
+  let(:json_response) { JSON.parse(response.body) }
+  let(:cname) { (account.search_only? ? work.to_solr.dig("account_cname_tesim")&.first : account.cname) }
+  let(:date_created) { "1992-12-31" }
+  let(:description) { "This is a test collection" }
+
+  let(:attributes) do
+    {
+      date_created: [date_created],
+      collection_type_gid: collection_type_gid,
+      description: [description],
+      title: ["Test Title"],
+      visibility: "open"
+    }
+  end
+
+  let(:collection) { Collection.create(attributes) }
+  let(:collection_branding) { CollectionBrandingInfo.new(collection_id: collection.id, role: "banner", filename: "nypl-hydra-of-lerna.jpg").save("spec/internal_test_hyku/spec/fixtures/images/nypl-hydra-of-lerna.jpg", false) }
```

Comment:
Will this file **fixtures/images/nypl-hydra-of-lerna.jpg"** in hyrax always be there.
Why did you not just give a string to local_path and then check the collection_banner_url matches it, so everything is with our control. What is the problem witrh passing it a local_path with a file path as string?

On calling new and then save rather than using either a factory or an instance double, we can keep that if you want

Reply:
We use that file in other tests, its kind of like our test file. Giving it a fake path results in a file not found error when doing it like this. I think you make a good point about everything being in our controll. Give me a few minutes to try a refactor

Reply:
Thanks @BertZZ